make/migrations/postgresql/0030_2.0.0_schema.up.sql

```diff
@@ -184,3 +184,43 @@ CREATE TABLE audit_log
  username       varchar(255) NOT NULL,
  op_time        timestamp default CURRENT_TIMESTAMP
 );
+
+/*migrate access log to audit log*/
+/*TODO drop table access_log?*/
+DO $$
+DECLARE
+    access RECORD;
+BEGIN
+    FOR access IN SELECT * FROM access_log
+    LOOP
+      /*insert project create and delete*/
+      IF access.operation = 'create' OR access.operation = 'delete' THEN
```

Comment:
The resources deleted can be project and image

Reply:
sure